packages/ckeditor5-dev-tests/bin/check-dependencies.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const glob = require( 'glob' );
+
+const cwd = process.cwd();
+const packageJson = require( path.join( cwd, 'package.json' ) );
+
+const currentPackageName = packageJson.name;
+const dependencies = Object.keys( packageJson.dependencies || {} );
+const devDependencies = Object.keys( packageJson.devDependencies || {} );
+
+const missingDependencies = new Set();
+const missingDevDependencies = new Set();
+const invalidFiles = new Set();
+
+const globPattern = path.join( cwd, '@(src|tests)/**/*.js' );
```

Comment:
This will also check `src/lib` which it should not (we keep there a lodash build).

Reply:
I have no idea how to write a glob pattern which:

- skips `src/lib/**/*.js`,
- accepts `src/file.js` and `src/dir/file.js`.

I had a problem with `src/file.js` and I decided to check the path in the `for...of` loop.